perses/rjmc/geometry.py

```diff
@@ -117,16 +117,22 @@ def propose(self, top_proposal):
 
                 #propose a bond and calculate its probability
                 bond = self._get_relevant_bond(atom, bond_atom)
-                r_proposed = self._propose_bond(bond, top_proposal.beta)
-                logp_r = self._bond_logp(r_proposed, bond, top_proposal.beta)
+                r_proposed = self._propose_bond(bond, beta)
+                bond_k = bond.type.k*units.kilocalorie_per_mole/units.angstrom**2
+                sigma_r = units.sqrt(1/beta*bond_k)
+                logZ_r = np.log((np.sqrt(2*np.pi)*sigma_r/sigma_r.unit))
+                logp_r = self._bond_logq(r_proposed, bond, beta) - logZ_r
 
                 #propose an angle and calculate its probability
                 angle = self._get_relevant_angle(atom, bond_atom, angle_atom)
-                theta_proposed = self._propose_angle(angle, top_proposal.beta)
-                logp_theta = self._angle_logp(theta_proposed, angle, top_proposal.beta)
+                theta_proposed = self._propose_angle(angle, beta)
+                angle_k = angle.type.k*units.kilocalorie_per_mole/units.angstrom**2
```

Comment:
The AMBER force constants may also be off by a factor of two.  Trace the call @swails uses here carefully to double-check:
https://github.com/ParmEd/ParmEd/blob/master/parmed/structure.py#L2053-L2054